src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -180,6 +159,27 @@ public SwitchParameter Examples
         [Parameter(ParameterSetName = "Parameters", Mandatory = true)]
         public string Parameter { set; get; }
 
+        /// <summary>
+        /// List of Component's to search on.
```

Comment:
Public comment for property should start with "Gets and sets ...". Below too.

Reply:
Done.